v22.2/import.md

```diff
@@ -35,13 +35,23 @@ Optimize import operations in your applications by following our [Import Perform
 
 ## Required privileges
 
-#### Table privileges
+### Table privileges
 
 The user must have the `CREATE` [privileges](security-reference/authorization.html#managing-privileges) on the target database.
 
-#### Source privileges
+### Source privileges
 
-{% include {{ page.version.version }}/misc/source-privileges.md %}
+The user that runs `IMPORT` to external storage does **not** require the `ADMIN` role in the following scenarios:
+
+- Amazon S3 and Google Cloud Storage using `SPECIFIED` credentials. Azure Storage is always `SPECIFIED` by default.
+- [Userfile](use-userfile-for-bulk-operations.html)
+
+{% include {{ page.version.version }}/misc/external-io-privilege.md %}
+
+The user that runs `IMPORT` to external storage **does** require the [`admin` role](security-reference/authorization.html#admin-role) in the following scenarios:
```

Comment:
Same as the comment in `destination-privileges.md` I don't know if we want to give the user the impression that there are still operations that only admin can run, because thats not the case anymore. `EXTERNALIOIMPLICITACCESS` governs who can and cannot run against an ExternalStorage.